pandas/tools/tests/test_util.py

```diff
@@ -391,6 +391,14 @@ def test_downcast(self):
             res = pd.to_numeric(data, downcast=downcast)
             tm.assert_numpy_array_equal(res, expected)
 
+        #check that 0 works as a unsigned downcast
+
+        data = [0, 1, 2, 3]
```

Comment:
Did you want me to just edit everything between line 327 and line 368 in that test to include 0's in the arrays? Trying it before and after, it fails before and passes after. That being said if you don't want me to touch it for some reason it's relatively easy to make a new one. Also is there a reason that 340 and 341 is redundant?